stack/indexer/deb/builder.sh

```diff
@@ -8,15 +8,14 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-set -ex
+set -e
```

Comment:
Why are you removing the -x?

Reply:
Removed the x argument to keep the same options as in the rest of the stack scripts